src/main/java/com/homeworks/homework20/service/SchedulerService.java

```diff
@@ -0,0 +1,41 @@
+package com.homeworks.homework20.service;
+
+import com.homeworks.homework20.model.User;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class SchedulerService {
+    private static final String CRON = "*/10 * * * * *";
```

Comment:
Please move CRON to the property file.
https://stackoverflow.com/questions/24306004/task-scheduling-using-cron-expression-from-properties-file